ci/envs/39-latest-conda-forge_no_fiona.yaml

```diff
@@ -16,7 +16,7 @@ dependencies:
   - pytest-xdist
   # - fsspec  # to have one non-minimal build without fsspec
   # optional
-  - matplotlib
+  - matplotlib>=3.6
```

Comment:
for some reason this was installing an too old version of matplotlib (not supported by pandas), and forcing a more recent one seems to do the job